CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # osquery Changelog
 
+<a name="5.0.2"></a>
+## [5.0.2](https://github.com/osquery/osquery/releases/tag/5.0.2)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.9.0...5.0.2)
+
+Representing commits from 21 contributors! Thank you all.
+
+osquery 5.0 is a tremendously exciting release.
+* We now install into /opt/osquery on macOS and Linux for better portability.
+* Our default and recommended installation for macOS uses an application bundle to support entitlement-based features.
+* We now use Endpoint Security APIs for various event-based tables on macOS (more to come in the future!)
```

Comment:
Specifically we can say:

"We now support process event monitoring using EndpointSecurity APIs on macOS (with more event types to come in the future!)"